AsyncResync.md

```diff
@@ -82,6 +85,9 @@ class Program
 	}
  }  
 ```
+It is not a concern that we are blocking on async here as we are in a console app or windows services where there will be no deadlocks, and an additional thread being used is acceptable. There is a language proposal to take away this boilerplate code in C# 7.1
```

Comment:
https://github.com/dotnet/csharplang/blob/master/proposals/async-main.md
Apparently not C# 7.1, still in proposal at the moment